release/ml_user_tests/ray-lightning/ray_lightning_user_test.py

```diff
@@ -3,7 +3,14 @@
 import time
 
 import ray
-from simple_example import main
+import sys
+
+# add a switch here because master and latest branch
+# diverge; this will be called in `release/release_tests.yaml`
+if len(sys.argv) > 1 and sys.argv[1] == "master":
+    from simple_example_master import main
+else:
+    from simple_example import main
```

Comment:
Can we move this check to `simple_example.main` and delay imports?

Something like:

```python
if use_master:
    # ray_lightning >= 0.3.0
    from ray_lightning import RayStrategy

    trainer = pl.Trainer(
        strategy=RayStrategy(num_workers=num_workers, use_gpu=use_gpu),
        max_steps=max_steps,
    )
else:
    # ray_lightning < 0.3.0
    from ray_lightning import RayPlugin

    trainer = pl.Trainer(
        plugins=[RayPlugin(num_workers=num_workers, use_gpu=use_gpu)],
        max_steps=max_steps,
    )
```

Reply:
If you want to make this a bit cleaner and support the case when we release `0.3.0`, you can use `packaging.version` to compare versions, though I'm not sure what the version number is for master.

Reply:
thanks @matthewdeng , changed! this is smart!